includes/embeds/class-amp-youtube-embed.php

```diff
@@ -64,6 +64,12 @@ public function shortcode( $attr ) {
 	}
 
 	public function oembed( $matches, $attr, $url, $rawattr ) {
+		// Use the standard oembed handler if it's a playlist
+		$parsed_url = wp_parse_url( $url );
+		if ( isset( $parsed_url['path'] ) && false !== strpos( $parsed_url['path'], 'playlist' ) ) {
+			return wp_oembed_get( $url );
```

Comment:
The Youtube class is an embed handler, rather than a sanitizer, and from what I can tell, [the embeds always run before the sanitizers](https://github.com/Automattic/amp-wp/blob/0.3.3/includes/class-amp-content.php#L35-L41). Am I getting that wrong?